includes/subscriptions/assets/js/product-page.js

```diff
@@ -0,0 +1,21 @@
+jQuery( function ( $ ) {
+	'use strict';
+
+	var wc_payments_product = {
+		init: function () {
+			// Workaround to prevent duplicate WCPay subscriptions when submit button is spammed in Chrome.
+			$( '#publish' ).on( 'click', function ( e ) {
+				if (
+					$( 'select#product-type option:selected' )
+						.val()
+						.includes( 'subscription' )
+				) {
+					$( this ).attr( 'disabled', true );
```

Comment:
This feels like a pretty heavy workaround - adding a new jQuery JS file just to prevent double-submit (in a single browser). 

- Should we log this issue with WooCommerce Core?
- Could this double-submit issue affect non-subscription products too? If not, why are subscription products unique?

Reply:
> Should we log this issue with WooCommerce Core?

That is a good question. I agree that preventing multiple clicks to a submit button should be a standard across and not just desirable for subscription specific product.